felix/dataplane/linux/int_dataplane.go

```diff
@@ -1544,14 +1544,21 @@ func (d *InternalDataplane) setUpIptablesBPF() {
 			Action: iptables.AcceptAction{},
 		})
 
-		if t.IPVersion == 6 && !d.config.BPFIpv6Enabled {
-			for _, prefix := range rulesConfig.WorkloadIfacePrefixes {
-				// In BPF mode, we don't support IPv6 yet.  Drop it.
-				fwdRules = append(fwdRules, iptables.Rule{
-					Match:   iptables.Match().OutInterface(prefix + "+"),
-					Action:  d.ruleRenderer.IptablesFilterDenyAction(),
-					Comment: []string{"To workload, drop IPv6."},
-				})
+		if t.IPVersion == 6 {
+			if !d.config.BPFIpv6Enabled {
+				for _, prefix := range rulesConfig.WorkloadIfacePrefixes {
+					// In BPF mode, we don't support IPv6 yet.  Drop it.
```

Comment:
Out of date comment.

Given we only have single stack right now, do we have code to block IPv4 packets when in IPv6 mode?  Think we should do that since we're not able to police them.

Reply:
yes. we only have ipv6 programs loaded and these will fail to parse ipv4 packets and thus they deny them.